src/components/CategoryFilter.js

```diff
@@ -3,7 +3,9 @@ import PropTypes from 'prop-types';
 
 function CategoryFilter(props) {
   const category = useRef();
-  const { filt } = props;
+  const cat2 = useRef();
+  // const { filt } = props;
+  console.log(cat2);//eslint-disable-line
```

Comment:
Let's remove `console.log`
No `console.log` committed to the repo. It's ugly, it kills performance. and it can make confidential data be visible in the browser tools.